specifications/cdsc-wg1-02.md

```diff
@@ -1 +1,1047 @@
+
+* `authorize` - Rendered as a button indicating to the user that the button will allow the user to authorize the Client in some fashion, such as a button saying "Authorize {client_name}".
+* `link` - Rendered as a button indicating to the user that the button will allow the user to link or connect the user's account to the Client, such as a button saying "Link your account with {client_name}".
+* `goto` - Rendered as a button indicating to the user that the button will allow the user to go to the Client's website, such as a button saying "Visit the {client_name} website".
+
+### 6.4. Modifying Client Settings <a id="clients-settings-modify" href="#clients-settings-modify" class="permalink">🔗</a>
+
+Clients may modify fields in the Client Settings API by sending an authenticated HTTPS `PATCH` request to the `cds_client_settings_api` endpoint with the body of the request formatted a JSON object.
+The fields included in JSON object are the fields the Client intends to update with the submitted fields' values.
+If a field is not included in the `PATCH` request, the Server MUST leave the field unmodified from its current value.
+
+The following are fields that MAY be included in the `PATCH` request, and modification MUST be supported by Servers:
+
+* `default_scope`
+* `profile_visibility`
+* `profile_description`
+* `profile_slug`
+* `profile_button_uri`
+* `profile_button_style`
+* `profile_contact_name`
+* `profile_contact_email`
+* `profile_contact_phone`
+* `profile_contact_website`
+
+Servers MUST reject requests with a `400 Bad Request` response when fields are submitted that are not able to be modified by the Client or the submitted values are invalid.
+For valid `PATCH` requests from Clients, Servers MUST respond with a `200 OK` or `202 Accepted` response with an updated JSON object of the complete current Client Settings object.
+
+If a Server needs to asynchronously review and approve changes to any submitted Client Settings object fields that have been submitted by the Client and are different from the current values, for valid modification requests the Server MUST respond with a `202 Accepted` response, which indicates that the submission was accepted but not fully saved as completed yet.
+Any fields that have not been synchronously updated as part of the request and response MUST remain in the response as their previous values, and the Server MUST add one or more entries to the [listed Client Updates](#clients-updates-list) for the modified fields that need to be asynchronously reviewed and approved.
+Clients MAY then use the [Client Updates API](#client-updates-api) to track the asynchronous review of the modification request.
+If all submitted fields have been synchronously updated as part of the response, Servers MUST respond with a `200 OK` response.
+
+## 7. Client Updates API <a id="client-updates-api" href="#client-updates-api" class="permalink">🔗</a>
```

Comment:
As discussed in the group sync, renaming from "Client Updates API" to something else would be more helpful, such as "Client Messaging API"